exporters/opensearch-exporter/pom.xml

```diff
@@ -179,6 +179,12 @@
       <artifactId>agrona</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>io.github.acm19</groupId>
+      <artifactId>aws-request-signing-apache-interceptor</artifactId>
+      <version>2.3.1</version>
+    </dependency>
```

Comment:
❌ Please remove the version from here and put the dependency with the version in the `dependencyManagement` section of the [parent pom](https://github.com/camunda/zeebe/blob/main/parent/pom.xml#L197)